client/src/views/users/MyContactDetailsPage.vue

```diff
@@ -213,26 +200,4 @@ onMounted(async () => {
 //     right: 5%;
 //   }
 // }
```

Comment:
Remove this if not useful anymore?
If this is still useful for the future, add a comment to explain why it is commented.